### ISSUE TITLE
Add Accessibility Statement to PDS Storefront

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#3065]
Scope
---
The Porsche Design System documentation page (Storefront) is part of the open source PDS and globally available. To adhere with legal requirements and to expose the overall status and commitment to accessibility, a so called "Accessibility Statement" should be added.

Out of Scope
---
- Don't list each and every accessibility issue we know on this page, it's more to show a general overview

Notes  
---
- Official explanation here: https://www.w3.org/WAI/planning/statements/ 
- PDS example: https://designsystem.porsche.com/latest/must-know/accessibility/statement
- Which contact (email) can we add to this page?

Acceptance criteria
---
- [ ] Accessibility Statement page with contact possibility is available

Subtasks
---
- [ ] Task
